build.gradle

```diff
@@ -226,6 +226,10 @@ project(':iceberg-core') {
     compile "com.fasterxml.jackson.core:jackson-databind"
     compile "com.fasterxml.jackson.core:jackson-core"
     compile "com.github.ben-manes.caffeine:caffeine"
+    compile 'commons-dbutils:commons-dbutils:1.7'
+    compile "org.hsqldb:hsqldb:2.5.1"
+    compile "mysql:mysql-connector-java:8.0.21"
+    compile "org.postgresql:postgresql:42.2.18"
```

Comment:
1. I only see the hsqldb used for testing, what are the use of mysql and postgres?
2. should mark as `testCompile` instead
3. versions should go to `versions.props`
4. have you checked if the license is okay for hsqldb?

Reply:
thought it will be nice to bundle common jdbc drivers, probably its not good idea removed mysql and postgres. checking point 4

Reply:
if it's only for testing, [sqlite](https://mvnrepository.com/artifact/org.xerial/sqlite-jdbc) is probably the safest choice

Reply:
changed to [h2dataabse](https://www.h2database.com) and added it to LICENSE file. its  [Mozilla Public License Version 2.0](https://www.h2database.com/html/license.html) it seems compatible once we include it in LICENSE file